aws/aws.go

```diff
@@ -843,6 +844,66 @@ func (s *awsOps) detachInternal(volumeID, instanceName string) error {
 	return err
 }
 
+func (s *awsOps) Expand(
+	volumeID string,
+	newSizeInGiB uint64,
+) (uint64, error) {
+	vol, err := s.refreshVol(&volumeID)
+	if err != nil {
+		return 0, err
+	}
+	currentSizeInGiB := uint64(*vol.Size)
+	if uint64(*vol.Size) >= newSizeInGiB {
```

Comment:
use `currentSizeInGiB` instead of `uint64(*vol.Size)`

Also we must fail the Expand request with error that volume is already at a larger size than the requested size. The caller should decide what to do with the error.